doc/howto/cluster_groups.md

```diff
@@ -27,6 +27,13 @@ To assign `server1` to the `gpu` group and also keep it in the `default` group,
 
     lxc cluster group assign server1 default,gpu
 
+To add a cluster member to a specific group, use `lxc cluster group add` command.
+This command adds a cluster member to the specified group.
+
+For example, to add `cluster:node1` to the `newgroup` group , use the following command:
+
+    lxc cluster group add cluster:node1 newgroup
```

Comment:
```suggestion
    lxc cluster group add server1 gpu
```

Reply:
Hi, @ru-fu made the changes as requested. Thanks